css/style.css

```diff
@@ -121,10 +129,608 @@
   order: 3;
   flex-grow: 0;
   margin: 12px 0;
-  list-style-type: none;
+  list-style: none;
 }
 
 .fa {
   color: white;
   text-decoration: none;
+  font-size: 25px;
+}
+
+.item-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 682px;
+}
+
+.recentWork {
+  position: static;
+  left: 0;
+  top: 0;
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 26px;
+  padding-top: inherit;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0;
+  padding-bottom: 40px;
+}
+
+.recentWork > img {
+  position: static;
+  width: 327px;
+  height: 0;
+  left: 0;
+  top: 107px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.first {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] You seem to have used this combination of stylings repeatedly in your CSS file. In keeping with the DRY principles, please remove these stylings into a different CSS class and then add that class as an attribute in your HTML tags.

For example:
**CSS**
```
.d-flex {
  display: flex;
}

.d-flex-col {
  flex-direction: column;
}
```
**HTML**
```
<div class="first d-flex d-flex-col">...</div>
```